src/jAlert.js

```diff
@@ -796,9 +803,11 @@
         'autofocus': false, //pass a selector to autofocus on it
 
         'onOpen': function(alert){ //on open call back. Fires just after the alert has finished rendering
+            toggleFocusTrap();
             return false;
         },
         'onClose': function(alert){ //fires when you close the alert
+            toggleFocusTrap();
             return false;
```

Comment:
Migrate this to where .onOpen and .onClose is called. I don't think the user needs the ability to disable/override it and it'll potentially cause a headache if someone overrides .onClose, but not .onOpen.

For onOpen, i think you can just add it to the array like the others.
For onClose, you'll have to call toggleFocusTrap() right next to where .onClose is called.